pinot-common/src/main/java/org/apache/pinot/common/function/FunctionDefinitionRegistry.java

```diff
@@ -29,20 +29,10 @@ private FunctionDefinitionRegistry() {
   }
 
   public static boolean isAggFunc(String functionName) {
```

Comment:
This function looks like just an alias of `AggregationFunctionType.isAggregationFunction`.  Can we just remove it and let caller to call `AggregationFunctionType.isAggregationFunction` directly?